src/main/java/cubicchunks/asm/mixin/core/common/MixinChunk_Cubes.java

```diff
@@ -135,17 +135,6 @@ private ExtendedBlockStorage getEBS_CubicChunks(int index) {
         return cube.getStorage();
     }
 
-    // getEntityList is unlikely to be called sequentially many times for the same cube, no caching
-    private ClassInheritanceMultiMap<Entity> getEntityList_CubicChunks(int index) {
```

Comment:
@Barteks2x in that case I still need two different functions which will be used only once.